tutorials/mistral/evaluate_rag--mistral.ipynb

```diff
@@ -77,7 +77,7 @@
    "metadata": {},
```

Comment:
<p>I don't think you can delete this import, can you?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
This did not come through from ReviewNB. I think you are deleting the import of `set_global_handler`.

Reply:
<p>It's done higher up</p>